bin/check-asg-instances-created.rb

```diff
@@ -57,41 +57,57 @@ class CheckAsgInstanceCreated < Sensu::Plugin::Check::CLI
          long:        '--critical_limit Critical Limit',
          description: 'Critical Limit for launching and terminated instances'
 
+  option :time_limit,
+         short:       '-t T',
+         long:        '--time Timelimit',
+         description: 'Time Limit for launching and terminated instances (default 1800s)'
+
   def asg
     @asg = Aws::AutoScaling::Client.new
   end
 
+  def ec2
+    @ec2 = Aws::EC2::Client.new
+  end
+
   def describe_activities(asg_group_name)
     asg.describe_scaling_activities(
       auto_scaling_group_name: asg_group_name.to_s
     )
   end
 
+  def describe_instance(instance_id)
+    ec2.describe_instances(
+      instance_ids: [instance_id]
+    )
+  end
+
   def run
-    warning = 3
-    critical = 4
-    result_launched = ''
+    warning = 2
```

Comment:
remove this (warning, critical, time) and make a default value for the option, I see no value in doing so, especially since you are checking later if its defined. something like this should work for what you want to accomplish: https://github.com/sensu-plugins/sensu-plugins-aws/blob/3fdc09864bb0f6985c6d6888adf9b1b53c9018e3/bin/check-elb-nodes.rb#L66-L67 I am also curious why the numbers are changing, what was the rationale that the existing defaults were not sane?